src/main/java/kostka/moviecatalog/service/DbMovieService.java

```diff
@@ -79,33 +79,33 @@ public Movie getMovie(final Long movieId) {
                 .collect(Collectors.toList());
     }
 
-    public List<MovieListDto> getTop5RatingMoviesForCaching() {
-        return this.getTop5RatingMoviesFromDB().stream()
+    public List<MovieListDto> getTop10RatingMoviesForCaching() {
```

Comment:
I like the iead, I'd suggest to use some SQL limit query.
Instead of loading lsit of data and then throwing uneeded items away (performace).

Reply:
agree, I will adapt that ;)